crates/re_renderer/shader/decodings.wgsl

```diff
@@ -8,18 +8,66 @@ fn decode_nv12(texture: texture_2d<u32>, coords: vec2i) -> vec4f {
     let uv_row = u32(coords.y / 2);
     var uv_col = u32(coords.x / 2) * 2u;
 
-    let y = max(0.0, (f32(textureLoad(texture, vec2u(coords), 0).r) - 16.0)) / 219.0;
-    let u = (f32(textureLoad(texture, vec2u(u32(uv_col), uv_offset + uv_row), 0).r) - 128.0) / 224.0;
-    let v = (f32(textureLoad(texture, vec2u((u32(uv_col) + 1u), uv_offset + uv_row), 0).r) - 128.0) / 224.0;
+    let y = f32(textureLoad(texture, vec2u(coords), 0).r);
+    let u = f32(textureLoad(texture, vec2u(u32(uv_col), uv_offset + uv_row), 0).r);
+    let v = f32(textureLoad(texture, vec2u((u32(uv_col) + 1u), uv_offset + uv_row), 0).r);
+
+    let rgb = set_color_standard(vec3f(y, u, v));
+
+    return vec4f(rgb, 1.0);
+}
+
+/// Loads an RGBA texel from a texture holding an YUY2 encoded image at the given screen space coordinates.
+fn decode_yuy2(texture: texture_2d<u32>, coords: vec2i) -> vec4f {
+    // texture is 2 * width * height
+    // every 4 bytes is 2 pixels
+    let uv_row = u32(coords.y);
+    // multiply by 2 because the width is multiplied by 2
+    var uv_col = u32(coords.x) * 2u;
+
+    var y = 0.0;
+    if coords.x % 2 == 0 {
+        // we're on an even pixel, so we can sample the first y value
+        y = f32(textureLoad(texture, vec2u(uv_col, uv_row), 0).r);
+    } else {
+        // we're on an odd pixel, so we need to sample the second y value
+        // we add 2 to the column to get the second y value
+        y = f32(textureLoad(texture, vec2u(uv_col + 2u, uv_row), 0).r);
```

Comment:
Didn't notice either at first, but this offset is wrong actually. We already muliplied the uv_col with 2, so it points at the Y value right now. I found out when adding yuy2 to the (former) nv12 python test script which also highlighted that there was still some stuff missing in `ImageEncoded`

To illustrate:
With that offset:
<img width="1019" alt="image" src="https://github.com/rerun-io/rerun/assets/1220815/17ea29d3-c488-4181-9bea-568bd0042244">


Without that offset:
<img width="1016" alt="image" src="https://github.com/rerun-io/rerun/assets/1220815/b1e0c37c-da81-4762-81e8-425a5176ca47">